build.gradle

```diff
@@ -208,6 +208,19 @@ task sarosServer(type: Copy, dependsOn: [
   into 'build/distribution/server'
 }
 
+task sarosLsp(type: Copy, dependsOn: [
+  ':saros.core:test',
+  ':saros.lsp:test',
+  ':saros.lsp:jar'
+  ]) {
+  group 'Delivery'
+  description 'Builds and tests all modules required by the Saros for VS Code'
+
+  from project(':saros.lsp').jar
+  into 'build/distribution/lsp'
+  into 'vscode/out'
```

Comment:
Currently just vscode/out is needed but according to the others and a later use I thought it would be good keeping that output directory. What do you think?

Reply:
Chose the distribution directory in favor of upcoming role of the lsp.
Build procedure of the vscode plugin might change to gradle anyways.